addons/hr/migrations/13.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,90 @@
+hr           / hr.employee              / image_128 (binary)            : NEW attachment: True, isrelated: related, stored
+hr           / hr.employee              / image_1920 (binary)           : NEW attachment: True, hasdefault
+hr           / hr.employee              / image_256 (binary)            : NEW attachment: True, isrelated: related, stored
+hr           / hr.employee              / image_512 (binary)            : NEW attachment: True, isrelated: related, stored
+hr           / hr.employee              / image_medium (binary)         : DEL attachment: True
+hr           / hr.employee              / image_small (binary)          : DEL attachment: True
+# DONE: pre-migration
+hr           / hr.plan                  / active (boolean)              : NEW hasdefault
+hr           / hr.plan                  / name (char)                   : NEW required
+hr           / hr.plan                  / plan_activity_type_ids (many2many): NEW relation: hr.plan.activity.type
+hr           / hr.plan.activity.type    / activity_type_id (many2one)   : NEW relation: mail.activity.type, hasdefault
+hr           / hr.plan.activity.type    / note (html)                   : NEW
+hr           / hr.plan.activity.type    / responsible (selection)       : NEW required, selection_keys: ['coach', 'employee', 'manager', 'other'], req_default: function, hasdefault
+hr           / hr.plan.activity.type    / responsible_id (many2one)     : NEW relation: res.users
+hr           / hr.plan.activity.type    / summary (char)                : NEW
+# NOTHING TO DO: new models
+hr           / res.company              / hr_presence_control_email_amount (integer): NEW
+hr           / res.company              / hr_presence_control_ip_list (char): NEW
+hr           / res.users                / barcode (char)                : previously in module base
+# NOTHING TO DO
+
+---XML records in module 'hr'---
+NEW hr.plan: hr.offboarding_plan (noupdate)
+NEW hr.plan: hr.onboarding_plan (noupdate)
+NEW hr.plan.activity.type: hr.offboarding_setup_compute_out_delais (noupdate)
+NEW hr.plan.activity.type: hr.offboarding_take_back_hr_materials (noupdate)
+NEW hr.plan.activity.type: hr.onboarding_plan_training (noupdate)
+NEW hr.plan.activity.type: hr.onboarding_setup_it_materials (noupdate)
+NEW hr.plan.activity.type: hr.onboarding_training (noupdate)
+NEW ir.actions.act_window: hr.hr_departure_wizard_action
+NEW ir.actions.act_window: hr.hr_employee_action_from_user
+NEW ir.actions.act_window: hr.hr_employee_public_action
+NEW ir.actions.act_window: hr.hr_plan_action
+NEW ir.actions.act_window: hr.hr_plan_activity_type_action
+NEW ir.actions.act_window: hr.plan_wizard_action
+NEW ir.actions.act_window: hr.res_users_action_my
+DEL ir.actions.act_window: hr.hr_employee_action_subordinate_hierachy
+NEW ir.actions.report: hr.hr_employee_print_badge [renamed from hr_attendance module]
```

Comment:
you should handle in this module the xmlid rename

Reply:
It's `renamed from` it could be not exist. Isn't it better done in `renamed to` side?

Reply:
if not exist, no problem, the xmlid rename don't break in that case

the rename always is done in destiny module. so if the rename is hr_attendance -> hr, then corresponds to hr module have he xmlid rename script.

Reply:
Thanks, done